docs/source/developers/guide/communication.rst

```diff
@@ -0,0 +1,77 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
+.. or more contributor license agreements.  See the NOTICE file
+.. distributed with this work for additional information
+.. regarding copyright ownership.  The ASF licenses this file
+.. to you under the Apache License, Version 2.0 (the
+.. "License"); you may not use this file except in compliance
+.. with the License.  You may obtain a copy of the License at
+
+..   http://www.apache.org/licenses/LICENSE-2.0
+
+.. Unless required by applicable law or agreed to in writing,
+.. software distributed under the License is distributed on an
+.. "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+.. KIND, either express or implied.  See the License for the
+.. specific language governing permissions and limitations
+.. under the License.
+
+.. _communication:
+
+*************
+Communication 
+*************
+
+**About the team**
+
+The team is full of experts, software engineers and core developers
+but also full of users, learners and enthusiasts that love doing what we
+do and we want to spread this enthusiasm to you also.
+
+We all have silly questions and we all need help lots of times.
+We encourage you to be open to communication and we will help as much as
+we can.
+
+Where to get help 👋
+====================
+
+Jira
+~~~~~~~~
+
+Thhe most used way is to communicate through Jira.
+When to use this option?
+
+- **Want to solve a bug** → post an issue on Jira and connect with the team
+  in the comments.
+- **Want to solve an issue that is already in Jira** → connect with the team
+  in the comments.
+
+Jira is used to discuss the problem before the PR is made (how to start,
+where to search for the solution, etc..).
+
+.. note::
+   How to make a Jira ticket. See `Tips for using JIRA <https://arrow.apache.org/docs/developers/contributing.html#tips-for-using-jira/>`_.
+
+Github
+~~~~~~
+
+- Github is used to discuss the **content of the PR**.
+- **You have a problem with the build** → you can make an issue on GitHub
+  and discuss the possible solution (or write to the user mailing list).
+
+  It is scary to make an issue for something that you have no clue about.
+  But no problem, we all have to go through that!
+
+  .. warning::
+     Make sure to add the version of the operating system and the software
+     you are using in the ticket description plus the debug information/error.
+
+User and dev mailing list
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can subscribe to the user or development mailing list and browse for
+previous topics or ask a question. There is also a biweekly developers sync
+call that anyone is welcome to join. It is held every second Wednesday at
+12:00 noon Eastern time. Link to the meeting is sent to the dev mailing list.
```

Comment:
I would not put the exact date and time in the documentation as that can always change, just mention that it's announced on the dev mailing-list.